crypto/dilithium/pqcrystals_dilithium_ref_common/sign.c

```diff
@@ -225,6 +238,27 @@ int crypto_sign_signature_internal(ml_dsa_params *params,
   /* FIPS 204: line 33 Write signature */
   pack_sig(params, sig, sig, &z, &h);
   *siglen = params->bytes;
+
+  /* FIPS 204. Section 3.6.3 Destruction of intermediate values. */
+  OPENSSL_cleanse(&n, sizeof(n));
+  OPENSSL_cleanse(seedbuf, sizeof(seedbuf));
+  OPENSSL_cleanse(rho, SEEDBYTES);
```

Comment:
This seems redundant with the cleansing of seedbuf which all of these point into. See comment above around cleansing the pointers.

Reply:
addressed in [dafaa7e](https://github.com/aws/aws-lc/pull/2017/commits/dafaa7e90387df5c2e880750012cc57ac1726bcf)